### ISSUE TITLE
dpkg error: trying to overwrite '/opt/ros/melodic/bin/RawLogViewer', which is also in package ros-melodic-mrpt1 1.5.9-1bionic.20210509.072520 dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)

### ISSUE THREAD

**Issue by SRai22** [mrpt-ros-pkg/mrpt_navigation#128]
When trying to perform rosdep install -y --from-paths . --ignore-src --rosdistro=melodic, tries to install both ros-melodic-mrpt1 and ros-melodic-mrpt2. Conflict arises for RawLogViewer and results in broken packages. 

Appreciate any help in this regard

**Comment by jlblancoc** [comment]
Mmm .. interesting, thanks! 
All recent updates were pushed to noetic, btw.

The correct solution should be removing mrptq (even from their repos, I'll try that) since all modern versions of all Ros packages now uses mrpt2 only. To start, remove mrptq and try again??

**Comment by SRai22** [comment]
I don't have a mrptq installed in the system. Also I changed all the CMakeLists to find mrpt 2. The error persists.

**Comment by jlblancoc** [comment]
I found this to be caused by `mrpt_bridge`. I already submitted an updated version to rosdistro for Melodic. In the meanwhile that pkg can be built from sources. 

Also, in this (mrpt_navigation) repo, there's code still to be ported to the latest mrpt2. 
Try installing the pkg ros-melodic-mrpt2, then building from sources. 

I just fixed and committed a few corrections, but more are pending. If you see easily how to fix more, feel free of opening PRs if you do it earlier than myself! ;-)
